tutorials/01_neural_networks.ipynb

```diff
@@ -0,0 +1,957 @@
+      "q_0: ┤ H ├┤ P(2.0*x[0]) ├──■────────────────────────────────────■────■──»\n",
+      "     ├───┤├─────────────┤┌─┴─┐┌──────────────────────────────┐┌─┴─┐  │  »\n",
+      "q_1: ┤ H ├┤ P(2.0*x[1]) ├┤ X ├┤ P(2.0*(π - x[0])*(π - x[1])) ├┤ X ├──┼──»\n",
+      "     ├───┤├─────────────┤└───┘└──────────────────────────────┘└───┘┌─┴─┐»\n",
+      "q_2: ┤ H ├┤ P(2.0*x[2]) ├──────────────────────────────────────────┤ X ├»\n",
+      "     └───┘└─────────────┘                                          └───┘»\n",
+      "«                                          ┌───┐»\n",
+      "«q_0: ──────────────────────────────────■──┤ H ├»\n",
+      "«                                       │  └───┘»\n",
+      "«q_1: ──────────────────────────────────┼────■──»\n",
+      "«     ┌──────────────────────────────┐┌─┴─┐┌─┴─┐»\n",
+      "«q_2: ┤ P(2.0*(π - x[0])*(π - x[2])) ├┤ X ├┤ X ├»\n",
+      "«     └──────────────────────────────┘└───┘└───┘»\n",
+      "«             ┌─────────────┐                                       »\n",
+      "«q_0: ────────┤ P(2.0*x[0]) ├────────────────────────────────────■──»\n",
+      "«             └─────────────┘              ┌───┐┌─────────────┐┌─┴─┐»\n",
+      "«q_1: ──────────────────────────────────■──┤ H ├┤ P(2.0*x[1]) ├┤ X ├»\n",
+      "«     ┌──────────────────────────────┐┌─┴─┐├───┤├─────────────┤└───┘»\n",
+      "«q_2: ┤ P(2.0*(π - x[1])*(π - x[2])) ├┤ X ├┤ H ├┤ P(2.0*x[2]) ├─────»\n",
+      "«     └──────────────────────────────┘└───┘└───┘└─────────────┘     »\n",
+      "«                                               »\n",
+      "«q_0: ──────────────────────────────────■────■──»\n",
+      "«     ┌──────────────────────────────┐┌─┴─┐  │  »\n",
+      "«q_1: ┤ P(2.0*(π - x[0])*(π - x[1])) ├┤ X ├──┼──»\n",
+      "«     └──────────────────────────────┘└───┘┌─┴─┐»\n",
+      "«q_2: ─────────────────────────────────────┤ X ├»\n",
+      "«                                          └───┘»\n",
+      "«                                               »\n",
+      "«q_0: ──────────────────────────────────■───────»\n",
+      "«                                       │       »\n",
+      "«q_1: ──────────────────────────────────┼────■──»\n",
+      "«     ┌──────────────────────────────┐┌─┴─┐┌─┴─┐»\n",
+      "«q_2: ┤ P(2.0*(π - x[0])*(π - x[2])) ├┤ X ├┤ X ├»\n",
+      "«     └──────────────────────────────┘└───┘└───┘»\n",
+      "«                                          \n",
+      "«q_0: ─────────────────────────────────────\n",
+      "«                                          \n",
+      "«q_1: ──────────────────────────────────■──\n",
+      "«     ┌──────────────────────────────┐┌─┴─┐\n",
+      "«q_2: ┤ P(2.0*(π - x[1])*(π - x[2])) ├┤ X ├\n",
+      "«     └──────────────────────────────┘└───┘\n"
+     ]
+    }
+   ],
+   "source": [
+    "# specify the feature map\n",
+    "fm = ZZFeatureMap(num_qubits, reps=2)\n",
+    "print(fm.draw())"
```

Comment:
Might want to look at the 'mpl' option since it gives nicer output in these notebookss

Reply:
done.